tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -495,24 +495,19 @@ where
 				) {
 					Ok(code) => {
 						debug!("create_identity_runtime {} OK", account_id_to_string(&who));
-						if let Some(key) = IdentityManagement::user_shielding_keys(&who) {
-							let id_graph =
-								ita_sgx_runtime::pallet_imt::Pallet::<Runtime>::get_id_graph(&who);
-							calls.push(OpaqueCall::from_tuple(&(
-								node_metadata_repo
-									.get_from_metadata(|m| m.identity_created_call_indexes())??,
-								SgxParentchainTypeConverter::convert(who),
-								aes_encrypt_default(&key, &identity.encode()),
-								aes_encrypt_default(&key, &code.encode()),
-								aes_encrypt_default(&key, &id_graph.encode()),
-							)));
-						} else {
-							add_call_from_imp_error(
-								calls,
-								node_metadata_repo,
-								IMPError::InvalidUserShieldingKey,
-							);
-						}
+						// For sure to get key here. It's already checked in pallet:
+						// ita_sgx_runtime::IdentityManagementCall::<Runtime>::create_identity
+						let key = IdentityManagement::user_shielding_keys(&who).unwrap();
```

Comment:
Even if we are pretty sure in the runtime that the key should exist, we should still avoid using `unwrap ()` (you never know where could go wrong - e.g. sidechain corruption). 

No panic in enclave is/will be a must.

Reply:
add check back.